README.md

```diff
@@ -354,13 +336,13 @@ Using `withOptions` will **not** modify the options initially provided, but retu
 
 ### Multiple batch import jobs
 
-  It is recommended to have one batch import job of a resource type at a given time. However in case you need to create multiple batch import jobs by calling `batchCreate` multiple times. The jobs may be concurrently executed. In this case that the server APIs may return rate limiting errors. It is recommended to adjust the `pollingInterval` option accordingly.
+  It is recommended to have one batch import job of a resource type at a given time. However in case you need to create multiple batch import jobs by calling `batchCreate` multiple times. The jobs may be concurrently executed. In this case that the server APIs may return rate limiting errors. It is recommended to adjust the `pollInterval` option accordingly.
```

Comment:
@patrick-fs this is for: https://fullstory.atlassian.net/browse/ECO-8825
it's actually a typo here, `pollInterval` is documented a line#323

Reply:
Cool, I see the options now.

Head's up that I reorganized a code sample a little bit with this commit: 83102ac8461bb0949eef02f4c61c641648408104

PTAL and feel free to drop it if you'd prefer not to merge the change

Reply:
Looks good. Thanks! Good to merge?